src/ws-header/ws-header.scss

```diff
@@ -65,6 +69,15 @@ $header-border-color: schema-color(4) !default;
   }
 }
 
+@mixin list-item() {
+  font-weight: bold;
+  color: $miroschka;
+  &.is-current {
```

Comment:
is-active would be a more standard name since we use it for every active state :)